stack-ghc-lib.yaml

```diff
@@ -1,12 +1,10 @@
-resolver: nightly-2022-01-10
+resolver: lts-19.20
 packages:
 - .
 extra-deps:
 - lsp-1.4.0.0
 - lsp-types-1.4.0.0
 - lsp-test-0.14.0.2
-- url: https://daml-binaries.da-ext.net/da-ghc-lib/ghc-lib-33b606cf39f957a5e75b7c3b44f18a0d.tar.gz
-- url: https://daml-binaries.da-ext.net/da-ghc-lib/ghc-lib-parser-33b606cf39f957a5e75b7c3b44f18a0d.tar.gz
```

Comment:
I removed these two lines because the URLs are no longer working. The build should still work with `ghc-lib{,-parser}` from the snapshot

According to git blame I added these lines in https://github.com/digital-asset/daml-ghcide/pull/19, but I'm not sure why. It might have been trying to override `ghc-lib{,-parser}` from the snapshot, but we only really need to do that in the daml repo.

Reply:
For now I switched the resolver to `lts-18.28` (GHC 8.10.7). This means that CI here no longer matches the way we build `daml-ghcide` in the daml repo (GHC 9.0.2 and `ghc-lib{,-parser}` generated with `--ghc-flavor=da-ghc-8.8.1`), so I opened https://github.com/digital-asset/daml/issues/14794.

Thanks for helping me figure this out @cocreature